### PR TITLE
task-3: moved code to custom cartridge

### PR DESCRIPTION
**Pull request by Vladzio26**

### CODE REVIEW

File: storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/resources/common.properties [hunk 1]
```diff
@@ -0,0 +1,79 @@
+minicart.viewcart=View Cart
```
Comment (oleksandra-davydchuk) [inline storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/resources/common.properties:1]: You don't need to copy all from .properties file, just new lines of code which you need in task.
.properties files are not overwritten, just supplemented

File: storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/default/product/productTileDiscount.isml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="row tile-body-footer">
+    <div class="ratings col-xs-12 col-sm-6">
+        <isset name="discount" value="${${(((finalOrigPrice.list.decimalPrice - price.decimalPrice)*100)/finalOrigPrice.list.decimalPrice).toFixed(0)}}" scope="page" />
+        <span style="color:red; text-transform:uppercase;">
+            ${Resource.msgf('label.bestDeal', 'common', discount, null)}
```
Comment (oleksandra-davydchuk) [inline storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/default/product/productTileDiscount.isml:5]: not correct
msgf

File: storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/resources/common.properties (hunk 1 above)
Comment (Vladzio26) [reply storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/resources/common.properties:1]: https://github.com/Vladzio26/sfcc-demo-task-review/pull/17/files
fixed here

File: storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/default/product/productTileDiscount.isml (hunk 2 above)
Comment (Vladzio26) [reply storefront-reference-architecture-master/cartridges/app_practical_tasks/cartridge/templates/default/product/productTileDiscount.isml:5]: https://github.com/Vladzio26/sfcc-demo-task-review/pull/17/files
fixed here

**Pull request merged**